packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -478,6 +488,20 @@ class EditableText extends StatefulWidget {
   final Color selectionColor;
 
   /// Optional delegate for building the text selection handles and toolbar.
+  ///
+  /// The [EditableText] widget used on its own will not trigger the display
+  /// of the selection toolbar by itself. The toolbar can be shown by
+  /// programmatically calling [EditableTextState.showToolbar] in response
+  /// to an appropriate user event.
+  ///
+  /// See also:
+  ///
+  ///  * [CupertinoTextField], which wraps a [EditableText] and which shows the
+  ///    selection toolbar upon user events that are appropriate on the iOS
+  ///    platform.
+  ///  * [TextField], a Material Design themed wrapper of [EditableText], which
+  ///    shows the selection toolbar upon appropriate user events based on the
```

Comment:
it shows a platform-specific toolbar too, right?

Reply:
It does. We don't necessarily have to talk about it here though.